crates/sound/src/device.rs

```diff
@@ -489,7 +467,7 @@ impl VhostUserSoundThread {
             }
         }
 
-        Ok(false)
+        Ok(true)
```

Comment:
Why this change is needed?

IIUC `handle_event()` should return `true` only if an exit event is received, but we are discussing removing that return value: https://github.com/rust-vmm/vhost/issues/144

I think we should check better that part into the code. It looks like we are not following that requirement.

Reply:
I did not meant to change this. I will fix it. Thanks.

Reply:
What do you mean with `I think we should check better that part into the code. It looks like we are not following that requirement.`?

Reply:
We are not currently using it and `handle_event()` is always returning `True`. How the returning value of `process_tx()` shall be used? What is an exit event?

Reply:
I don't know how it can be used, honestly ;-) I think we can just remove it if it is not used and re-introduce when we need it. (maybe the idea was to return `false` if we don't have any more work to do? sorry, I don't remember)

About the `exit_event` is the eventfd we use to tell the event loop we want to exit: https://github.com/virtio-sound/vhost-device/blob/cbc80bee229ed0b827d45ef14d3b89d9396d72f8/crates/sound/src/device.rs#L569

It seems now it is directly handled by the backend loop in vhost-user-backend, so the event should not be propagated to our `handle_event`.
Maybe it was an old design. I think we will remove the return value. I'll look into it next week (for the vhost-user-backend point of view).

Reply:
I see, I'll address the returning value for those functions in a different PR. I just created #39 to not forget. Thanks.